packages/core/package.json

```diff
@@ -7,9 +7,6 @@
   "scripts": {
     "depcheck": "depcheck",
     "pretest": "npm run lint && npm run depcheck",
-    "redis": "docker rm --force redis-commerce-sdk; docker run -dp 6379:6379 --name redis-commerce-sdk redis:5-alpine",
```

Comment:
I'm not sure we want to remove these from here since I added them to aid in development and the redis code is in this package.

Reply:
Moved it back